src/main/java/org/corfudb/cmdlets/corfu_layout.java

```diff
@@ -35,6 +38,7 @@
 public class corfu_layout implements ICmdlet {
 
     private static Map<String, NettyClientRouter> routers = new ConcurrentHashMap<>();
+    private static Map<String, LayoutView> layoutViews = new ConcurrentHashMap<>();
```

Comment:
Why is this cmdlet storing layout views? It shouldn't be dealing with layout views, it is supposed to "directly interact with a layout server".